Firmware/Filament_sensor.cpp

```diff
@@ -128,7 +128,7 @@ void Filament_sensor::triggerFilamentRemoved() {
     if (runoutEnabled
         && (eFilamentAction == FilamentAction::None)
         && (
-            moves_planned() != 0
+            e_active()
             || printJobOngoing()
```

Comment:
Looking back at this... shouldn't this be `e_active() && printJobOngoing()`? I don't know of any is situation where we can recover from filament runout except when printing.

Reply:
Is first layer calibration a `printJobOngoing()`?

Reply:
No it's only true if `card.sdprinting = true`, which happens only when a gcode file is printing. Or when `usb_timer.running()` is `true`, which only happens if the user is sending commands via Serial

Reply:
Filament runout during First Layer Calibration is ignored explicitly. Similar to how we disable it when MMU is enabled.
![image](https://github.com/prusa3d/Prusa-Firmware/assets/8218499/4f60dbbb-bffb-4933-9694-57cb6a170aef)